docs/en/04_Changelogs/4.8.0.md

```diff
@@ -10,47 +10,58 @@
 
 ### Support for silverstripe/graphql v4 {#graphql-v4}
 
-The [silverstripe/graphql](http://github.com/silverstripe/silverstripe-graphql/issues) module
-to drive various parts of the CMS UI, as well as building your own GraphQL APIs in Silverstripe projects.
+The [silverstripe/graphql](http://github.com/silverstripe/silverstripe-graphql/issues) module is used
+to drive various parts of the CMS UI, as well as building your own GraphQL APIs in your Silverstripe CMS projects.
 
-The CMS currently ships with `silverstripe/graphql:^3`, which is stable but slow for larger schemas.
-The module was rewritten from scratch with performance in mind
-(see [goals](https://github.com/silverstripe/silverstripe-framework/issues/8824) and [pull request](https://github.com/silverstripe/silverstripe-graphql/pulls)).
-The resulting`silverstripe/graphql:^4` release is currently in a alpha stage.
-We encourage everyone to get started, read our new [GraphQL documentation](https://docs.silverstripe.org/en/4/developer_guides/graphql/), 
-provide [feedback](https://github.com/silverstripe/silverstripe-graphql/issues) and help us stabilise the module.
+The CMS currently ships with `silverstripe/graphql:^3.5`, which is stable but slow for larger schemas.
+The module was rewritten from scratch with performance in mind.
 
-In order to facilitate the new module release, most GraphQL helper classes to drive the CMS UI
-have been marked as deprecated, and moved to `GraphQL/_legacy` folders within core modules.
-The GraphQL schema under `admin/graphql` has been altered slightly to enable forwards compability,
-with lower-camelcase field names, and a different filter argument structure.
-In the unlikely case that you are relying on those classes or queries directly,
-consider upgrading to `silverstripe/graphql:^4`.
-Your own GraphQL schemas outside of `admin/graphql` should not be affected by the changes
-if you choose to stay on `silverstripe/graphql:^3` for the time being.
```

Comment:
This part seems to have a lot of overlap with "GraphQL 3 code moved to _legacy folder", so I moved it there